tests/rptest/tests/license_upgrade_test.py

```diff
@@ -136,13 +136,19 @@ def test_license_upgrade(self):
             return
 
         # Upload a license
-        self.admin.put_license(license)
+        assert self.admin.put_license(license).status_code == 200
```

Comment:
future reference, the `raise_for_status` on a response object is a handy way to do this